python/tests/test_adjoint_solver.py

```diff
@@ -1152,6 +1152,99 @@ def test_periodic_design(self):
 
             print(f"PASSED: filter function = {selected_filter.__name__}")
 
+    def test_anisotropic_resolution(self):
```

Comment:
Suggest renaming to `test_nonuniform_grid_resolution` to be more precise and avoid confusion with more common usage of anisotropic to refer to ε.

Reply:
Your suggested name is clearer, but it still seems misleading because nonuniform resolution may also refer to something else, such as the case below. Shall we use a longer name such as `test_unequal_horizontal_vertical_resolution`?
![image](https://github.com/NanoComp/meep/assets/60903466/88a3319d-bdef-442a-a421-9fea5c5f7fac)